maps-compose/src/main/java/com/google/maps/android/compose/CameraPositionState.kt

```diff
@@ -27,12 +27,7 @@ import com.google.android.gms.maps.GoogleMap
 import com.google.android.gms.maps.Projection
 import com.google.android.gms.maps.model.CameraPosition
 import com.google.android.gms.maps.model.LatLng
-import kotlinx.coroutines.CancellableContinuation
-import kotlinx.coroutines.CancellationException
-import kotlinx.coroutines.Job
-import kotlinx.coroutines.cancel
-import kotlinx.coroutines.currentCoroutineContext
-import kotlinx.coroutines.suspendCancellableCoroutine
+import kotlinx.coroutines.*
```

Comment:
Nit: Use single name imports here. I have my preferences configured to the following:

![image](https://user-images.githubusercontent.com/463186/175344584-d52ebae1-1d61-4305-98f0-b16a05f59912.png)

This should probably be enforced in the linter.

Reply:
Ah, good catch - I fixed the imports to not use wildcards and updated my AS settings.